modules/standard/online/log_controller.py

```diff
@@ -1,56 +1,73 @@
-from core import db, command_request
-from core.command_param_types import Any
+from core.command_param_types import Any, Const
 from core.command_request import CommandRequest
 from core.db import DB
-from core.decorators import instance, command, event
+from core.decorators import instance, command
+
 
 @instance()
 class LogController:
 
     def inject(self, registry):
-        self.bot = registry.get_instance("bot")
         self.db: DB = registry.get_instance("db")
-        self.text = registry.get_instance("text")
-        self.util = registry.get_instance("util")
-        self.pork_service = registry.get_instance("pork_service")
-        self.character_service = registry.get_instance("character_service")
-        self.discord_controller = registry.get_instance("discord_controller")
-        self.command_alias_service = registry.get_instance("command_alias_service")
-        self.alts_service = registry.get_instance("alts_service")
-        self.alts_controller = registry.get_instance("alts_controller")
-
-    @command(command="logon", params=[Any("logon_message")], access_level="member",
-             description="Sets your own custom logon message")
+
+    @command(command="logon", params=[], access_level="member", description="Check your current logon message")
+    def check_current_logon(self, request):
+        current_logon = self.get_logon(request.sender.char_id)
+        if current_logon:
+            return "%s's login message is: %s" % (request.sender.name, current_logon)
```

Comment:
one more thing, let's change login to logon to be consistent here

Reply:
Woah how did you even spot that! 🔍